integr8ly/integreatly-operator
# MGDAPI-4399 update the docs links and structure

# Issue link
https://issues.redhat.com/browse/MGDAPI-4399 && https://issues.redhat.com/browse/MGDAPI-4393

# What
Updated docs structure, moved some docs from help repo, fixed broken links

# Verification steps
Locally run make mkdocs/serve and verify that the new layout makes sense and that links are working
